### DISCUSSION TITLE
404 There isn't a GitHub Pages site here.

### DISCUSSION THREAD

**Discussion by Abdulkadirc99** [community/community#30398] [Other Feature Feedback, Questions, & Ideas]
So i've been following a course that was recorded in 2018 and was wondering if any changes have occurred since then. i l know this was asked before but no solution worked for me so far.
my website: https://github.com/Abdulkadirc99/Abdulkadir.github.io
when i try to access : Abdulkadir.github.io i get the 404 error.
i tried the course solution by adding a file path: Abdulkadir.github.io/forms.html and still the same error.
the repository is public and followed every step correctly, so why is it not working?

**Comment by ichenglin** [comment]
Your **Github page repository** name **doesn't match with your username**,
it should be **`Abdulkadirc99.github.io`** but you named it **`Abdulkadir.github.io`**.

**Reply by Abdulkadirc99** [reply]
i still get the 404 error. but this time when add the file https://abdulkadirc99.github.io/forms.html it says file not found. but it's in the folder.

**Reply by ichenglin** [reply]
I just checked your repository,
and realize that you have your **`forms.html`** file **stored inside** a **`begin/`** folder,
which explains why it **fails the find the file**.

If you wish to access the **HTML file** with link **`https://Abdulkadirc99.github.io/forms.html`**,
you would have to upload the **HTML file** at the **root of the repository**,
in other words **outside** the **`begin/`** folder.
